src/main/java/io/bf2/kafka/common/Config.java

```diff
@@ -134,7 +139,7 @@ public class Config {
      * in the cluster. If this property is not specified, a default of {@link #DEFAULT_MAX_PARTITIONS}
      * will be used in this class.
      */
-    public static final String MAX_PARTITIONS = "max.partitions";
+    public static final String MAX_PARTITIONS = CREATE_TOPIC_POLICY_PREFIX + "max.partitions";
```

Comment:
I guess this is a bug, right? @grdryn

Reply:
I don't think it's necessarily a bug. I think it has always been an unprefixed property name here, and I didn't change that when changing the prefixes.

Maybe it makes sense to change it, but maybe not. I think this config property might be used for other things like metrics or the admin API server, so if we're going to add the prefix here, then we might need to set both versions (and keep both versions in sync over time). I might be wrong about that.

Reply:
If we are leaving it without a prefix we need to update the readme to match.

Reply:
This property is also used by MetricsManager in kas-fleetshard, so keeping it without the plugin specific prefix is required.,

Reply:
I see. change reverted. Thanks.